clang/docs/ReleaseNotes.rst

```diff
@@ -548,6 +548,11 @@ Miscellaneous Clang Crashes Fixed
 - Fixed internal assertion firing when a declaration in the implicit global
   module is found through ADL. (GH#109879)
 
+Leaf Functions Do Not Retain FP Bug Fixed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Fixed ``-fno-omit-frame-pointer`` leaf functions do not retain fp.(#GH108019)
```

Comment:
Put this under "Target-specific changes".

Maybe mention `-momit-leaf-frame-pointer` here as well.

Reply:
OK，Could you kindly review the changes I've made? Thank you.